compiler/frontend/src/org/jetbrains/kotlin/diagnostics/rendering/DefaultErrorMessages.java

```diff
@@ -1067,6 +1067,8 @@ public static DiagnosticRenderer getRendererForDiagnostic(@NotNull UnboundDiagno
         MAP.put(CATCH_PARAMETER_WITH_DEFAULT_VALUE, "Catch clause parameter may not have a default value");
 
         MAP.put(AMBIGUOUS_ANONYMOUS_TYPE_INFERRED, "Right-hand side has anonymous type. Please specify type explicitly", TO_STRING);
+        MAP.put(APPROXIMATED_LOCAL_TYPE_WILL_BECOME_NULLABLE,
```

Comment:
The reason I leaned towards an error is because the impact is not just big, but also sometimes non-obvious due to overloads. If people ignore this message in 1.8, then in 1.9 the code might still compile, but in an unexpectedly different way. But I'm not sure about this.

Reply:
Well, whatever. I guess it's their problem if they ignore warnings. Changed to a warning and added a link to the LC proposal.